docker-build/scripts/00_setup.sh

```diff
@@ -9,7 +9,11 @@ STRIP_SYMBOLS=1
 
 MYARCHTUNE=""
 MYCFLAGS=""
-MYRWDIFLAGS="-O3 -g -DNDEBUG"
+MYBUILDTYPE="RELWITHDEBINFO"
+RELWITHDEBINFOFLAGS="-O3 -g -DNDEBUG"
+DEBUGFLAGS="-O3 -g -DDEBUG -D_DEBUG -DNO_CATCH_EXCEPTIONS"
+RELEASEFLAGS="-O3 -g -DNDEBUG"
```

Comment:
Release the same as rel with debug info? At least i would set -g1 instead of full debug info.